src/event.cpp

```diff
@@ -128,6 +128,7 @@ wcstring event_get_desc(const event_t &e) {
             if (e.param1.pid > 0) {
                 result = format_string(_(L"exit handler for process %d"), e.param1.pid);
             } else {
+                // In events, PGIDs are stored as negative PIDs
```

Comment:
As we don't use that convention anywhere else I thought it was worth documenting.